### PR TITLE
Modify PPP

### PR DESCRIPTION
**Pull request by imanamirshah**

### CODE REVIEW

File: docs/team/imanamirshah.md [hunk 1]
```diff
@@ -43,7 +45,7 @@ stock
   request be given.
 
 
-- **Code Contributed:** [RepoSense link](#https://nus-cs2113-ay2324s2.github.io/tp-dashboard/?search=imanamirshah&breakdown=true&sort=groupTitle%20dsc&sortWithin=title&since=2024-02-23&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
```
Comment (nur-haziq) [inline docs/team/imanamirshah.md:46]: Good catch! Link works on my end 😄

**Review by nur-haziq — APPROVED** [review]
LGTM. Thanks for the changes.

File: docs/AboutUs.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+| <img src="https://assets.mycast.io/characters/mayor-3940618-normal.jpg" alt="Yi Hao image" width="100"/>                                           | Wong Yi Hao |   [Github](https://github.com/YHWong20)   |   [Portfolio](./team/yhwong20.md)   |
+| <img src="https://comicvine.gamespot.com/a/uploads/scale_medium/11/111746/5814382-powerpuffgirls-bubbles_large.png" alt="Iman image" width="100"/> |  Syed Iman  | [Github](https://github.com/imanamirshah) | [Portfolio](./team/imanamirshah.md) |
+| <img src="https://www.writeups.org/wp-content/uploads/Blossom-Powerpuff-Girls-Profile.jpg" alt="Haziq Image" width="100"/>                         |  Nur Haziq  |  [Github](https://github.com/nur-haziq)   |  [Portfolio](./team/nur-haziq.md)   |
+| <img src="https://i.pinimg.com/474x/24/f5/c4/24f5c491ea60062966abffacfdd5a13d.jpg" alt="Kota Image" width="100"/>                                  |    Kota     |    [Github](https://github.com/nkotaa)    |    [Portfolio](./team/nkotaa.md)    |
```
Comment (nkotaa) [inline docs/AboutUs.md:8]: I see I am Buttercup now.

**Review by nkotaa — APPROVED** [review]
Approved.

**Pull request merged**